### PR TITLE
Psl 194 convert pastel gtest

### PR DESCRIPTION
**Pull request by tanlemin91**

### CODE REVIEW

File: src/Makefile.gtest.include [hunk 1]
```diff
@@ -86,12 +88,13 @@ pastel_gtest_SOURCES +=\
...
 if ENABLE_WALLET
 pastel_gtest_SOURCES += \
+	wallet/gtest/test_accounting.cpp\
 	wallet/gtest/test_wallet.cpp\
 	wallet/gtest/test_wallet_ismine.cpp
 endif
 
 
-pastel_gtest_CPPFLAGS = $(AM_CPPFLAGS) -DBINARY_OUTPUT -DCURVE_ALT_BN128 -DSTATIC $(BITCOIN_INCLUDES) -I$(srcdir)/gtest
+pastel_gtest_CPPFLAGS = $(AM_CPPFLAGS) -DBINARY_OUTPUT -DCURVE_ALT_BN128 -DSTATIC $(BITCOIN_INCLUDES) -I$(srcdir)/gtest -I$(builddir)/test/
```
Comment (akobrin1) [inline src/Makefile.gtest.include:97]: remove that include, is idea is to remove any references to bitcoin_test

File: src/alertkeys.h [hunk 2]
```diff
@@ -3,8 +3,8 @@
...
-const char* pszTestNetPrivKey = "000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000";
+static const char* pszPrivKey = "000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000";
```
Comment (akobrin1) [inline src/alertkeys.h:6]: make it "constexpr auto pszPrivKey = "

**Review by akobrin1 — CHANGES_REQUESTED** [review]
remove bitcoin_test include

File: src/Makefile.gtest.include (hunk 1 above)
Comment (tanlemin91) [reply src/Makefile.gtest.include]: Fixed. I removed that include and add the raw data generation in Makefile.gtest.include

File: src/alertkeys.h (hunk 2 above)
Comment (tanlemin91) [reply src/alertkeys.h]: Fixed. Thanks

**Review by akobrin1 — APPROVED** [review]

**Pull request merged**